db/migrate/20170409170753_add_last_webfingered_at_to_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastWebfingeredAtToAccounts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :accounts, :last_webfingered_at, :datetime, null: true, default: nil
```

Comment:
Also modeled after subscription_expires_at (and db/migrate/20160919221059_add_subscription_expires_at_to_accounts.rb), I'm fine with timestamp if it's better